packages/fxa-settings/src/pages/Signin/SigninUnblock/container.tsx

```diff
@@ -62,7 +61,7 @@ const SigninUnblockContainer = ({
   const { email, hasLinkedAccount, hasPassword } = location.state || {};
 
   const wantsTwoStepAuthentication =
-    isOAuthIntegration(integration) && integration.wantsTwoStepAuthentication();
+    integration.isOAuth() && integration.wantsTwoStepAuthentication();
```

Comment:
Note that in this case isOAuth doesn't need to be a typeguard. It's just being used like a regular boolean.